unlock-app/src/components/interface/Error.js

```diff
@@ -23,13 +23,13 @@ export const Error = ({ children, error, close }) => {
   )
 }
 
-const mapStateToProps = ({ error }) => ({
-  error: error,
+const mapStateToProps = ({ errors }) => ({
+  error: errors.length ? errors[errors.length - 1] : null,
```

Comment:
Maybe a quick comment to indicate that since the UI only supports a single error for now we show the first one?